index.html

```diff
@@ -278,6 +278,18 @@ <h3 class="habs-frame-title-h3"> Skills</h3>
                 </li>
             </ul>
         </section>
+        <section id="contact" >
+            <h2 class="contact-title">Contact me</h2>
+            <p>If you have an application you are interested in developing, a feature that you need built or a project that needs coding. I’d love to help with it </p>
+            <form action="https://formspree.io/f/mdojgvgv" method="post">
+                <input type="text" placeholder="Yeremias" name="user_name" maxlength="30" required/>
+                <input type="email" placeholder="natajayanj14@gmail.com" name="user_mail" required/>
+                <textarea name="user_message" placeholder="Write your message here" maxlength="500" required></textarea>
+                <button type="submit" name="submit" > 
+                    <span> Get in touch </span> 
```

Comment:
- [ ] I suggest you remove `span` tag from inside `button` tag. there is no need for that, the button tag can handle the submission.